doc/source/packaging/diagrams/src_structure_diagram.rst

```diff
@@ -0,0 +1,64 @@
+.. _src structure diagram:
+.. graphviz::
+    :caption: Generic structure for the src/ directory
```

Comment:
@jorgepiloto Can literal tagging for "src/" be used in a caption. I'm guessing the answer is no, but I thought I'd ask.

Reply:
In principle, the "/" gets rendered without errors or any Sphinx warnings. I also tried to use "src\/" to escape the character and still showed the "/" symbol.